packages/modules/src/modules/banners/globalNewYear/GlobalNewYearBanner.tsx

```diff
@@ -0,0 +1,9 @@
+import { bannerWrapper, validatedBannerWrapper } from '../common/BannerWrapper';
+import getGlobalNewYearBanner from './components/getGlobalNewYearBanner';
+
+const GlobalNewYearBanner = getGlobalNewYearBanner();
```

Comment:
It doesn't really matter here, but usually we use this `getComponentX` pattern when we're running a design AB test or if there's some other reason that we'd want different versions of the component without having to change `BannerProps`. As we've just got the one version of this component, it's not super necessary - but everything will still work perfectly so we can leave this if you'd like!

Reply:
Yeah, I adapted this from the US EOY banner as a jumping off point, so this is a hangover from that.